core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2368,4 +2368,9 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
 
    @Message(value = "Cache '%s' has number of owners %d but is missing too many members (%d/%d) to reinstall topology", id = 694)
    MissingMembersException missingTooManyMembers(String cacheName, int owners, int missing, int total);
+
+   @Message(value = "Embedded queries are not supported without adding any Infinispan query module to the classpath." +
```

Comment:
"Query module not found. Add `infinispan-query` or `infinispan-query-core` to the classpath"